src/pages/evm/swap/confirm.svelte

```diff
@@ -0,0 +1,98 @@
+<script lang="ts">
+    import {Asset} from '@greymass/eosio'
+    import {getAddress} from 'ethers/lib/utils'
+    import Button from '~/components/elements/button.svelte'
+
+    import {evmAccount, activeSession} from '~/store'
+    import type {Token} from '~/stores/tokens'
+
+    export let from: Token
+    export let to: Token
+    export let amount: string
+    export let handleConfirm: () => void
+    export let handleBack: () => void
+</script>
+
+<style type="scss">
+    .container {
+        width: 100%;
+        margin: auto;
+        padding: 3em;
+        background-color: transparent;
+        text-align: center;
+        border: 2px solid var(--main-grey);
+
+        .top-section {
+            margin-bottom: 2em;
+        }
+
+        table {
+            width: 100%;
+            margin-bottom: 2em;
+
+            tr {
+                display: flex;
+                justify-content: space-between;
+                font-size: 11px;
+                border-top: 1px solid #ddd;
+
+                &:first-child {
+                    border: none;
+                }
+
+                td {
+                    padding: 1.5em;
+                }
+            }
+        }
+
+        .bottom-section {
+            max-width: 300px;
+            margin: auto;
+        }
+
+        /* Media Query for Mobile */
+        @media screen and (max-width: 768px) {
+            padding: 1em;
+
+            table {
+                tr {
+                    flex-direction: column;
+                    align-items: start;
+                }
+            }
+
+            .bottom-section {
+                max-width: 100%;
+            }
+        }
+    }
+</style>
+
+<div class="container">
+    <div class="top-section">
+        <h1>Transfer</h1>
+        <h3>Review and Sign</h3>
+    </div>
+
+    <table>
+        <tr>
+            <td>From {from.name}</td>
+            <td>{from?.name === 'EOS (EVM)' ? $evmAccount?.address : $activeSession?.auth.actor}</td
+            >
+        </tr>
+        <tr>
+            <td>To {to.name}</td>
+            <td>{from?.name === 'EOS' ? $evmAccount?.address : $activeSession?.auth.actor}</td>
+        </tr>
+        <tr>
+            <td>Amount</td>
+            <td>{Asset.from(Number(amount), '4,EOS')}</td>
```

Comment:
I know it's not part of the design (yet), but we should show the fee amount on this screen.

It can be retrieved from this table:

https://bloks.io/account/eosio.evm?loadContract=true&tab=Tables&account=eosio.evm&scope=eosio.evm&limit=100&table=config

And is the `ingress_bridge_fee` value (currently 0.0100). 

We can change this row into :

- "Deposit Amount" (the amount they entered)
- "Fee Amount" (the value of that table)
- "Received Amount" (the amount they entered minus the fee)

This way they'll know what their balance will be after the transaction

Reply:
I don't think that `ingress_bridge_fee` has the correct fee amount 🤔 I just did two transactions and the both cost around 0.003 EOS in fees.

Reply:
I wonder if the `ingress_bridge_fee` is just held the first time you make a transfer 🤔

Reply:
Ahh it looks like the `ingress_bridge_fee` is only applied when transferring from Native to EVM.